contracts/deposit-contract/depositContract.v.py

```diff
@@ -1,20 +1,24 @@
 ## compiled with v0.1.0-beta.7 ##
 
-MIN_DEPOSIT: constant(uint256(wei)) = as_wei_value(1, "ether")
-MAX_DEPOSIT: constant(uint256(wei)) = as_wei_value(32, "ether")
```

Comment:
Can you revert this change? it is easier to understand 32 ether than it is to understand the gwei amounts

Reply:
This is actually part of the spec, where gwei is to be used as the primary unit of account. Referenced here: https://github.com/ethereum/eth2.0-specs/pull/460

Reply:
Right, but you are using `as_wei_value` with the unit specified. So this change just adds unnecessary cognitive friction. 

 `as_wei_value(32, "ether") == as_wei_value(32000000000, "gwei")` so why change it?